up

```diff
@@ -1,56 +1,9 @@
-#!/bin/sh
-
+#!/bin/bash
 set -eu
 
-: "${DATALAD_REGISTRY_INSTANCE_PATH:=$PWD/instance}"
-export DATALAD_REGISTRY_INSTANCE_PATH
-: "${DATALAD_REGISTRY_DATASET_CACHE:=$DATALAD_REGISTRY_INSTANCE_PATH/cache}"
```

Comment:
@yarikoptic The `docker-compose.yml` file sets `DATALAD_REGISTRY_DATASET_CACHE` to `/app/instance/cache`.  Under what circumstances would you want to change that?

The Postgres password is currently hardcoded into `docker-compose.yml`.  Unless we're going to be using the Docker Compose setup in production (Are we?), I see no point in making it customizable.

Reply:
> Under what circumstances would you want to change that?

I think I might be doomed to change it for a production deployment eventually since it would need to reside on a different partition (no need to backup that cache, will be really inodes heavy etc -- might want to place it under a dedicated btrfs volume).  But I guess I could try orchestrate for that "outside" (at the filesystem level etc), and indeed let's just keep it to `instance/cache` for now.

> .. Docker Compose setup in production (Are we?) ...

I did think to use docker compose setup for production (and that is partially why wanted to ensure that processes do not run as root) -- makes it easier to ensure consistent installation even when switching the host etc.